### PR TITLE
Fixed TypeError in source cog's print function

### PR DESCRIPTION
**Pull request by ellieswaim**

### CODE REVIEW

File: bot/cogs/source_code_cog.py [hunk 1]
```diff
@@ -181,7 +181,7 @@ def chunk_iterable(self, iterable, chunk_size):
     
     def process_source(self, source: str, line_start: int = None, line_stop: int = None):
         split_source = [f'{i:03d} |  {value}' for i, value in enumerate(source.splitlines())]
-        filtered_source = split_source[line_start or 0: line_stop or len(source)]
+        filtered_source = split_source[line_start or 0: line_stop+1 if line_stop is not None else len(source)]
```
Comment (Jay-Madden) [inline bot/cogs/source_code_cog.py:184]: this can just be

if not line_stop else

Comment (ellieswaim) [reply bot/cogs/source_code_cog.py:184]: gotcha, i'll change it. also working on making lines print from 1 instead of 0.

Comment (ellieswaim) [reply bot/cogs/source_code_cog.py:184]: when i change it to if not line_stop else it gives me the TypeError again

File: bot/cogs/source_code_cog.py [hunk 2]
```diff
@@ -180,8 +180,12 @@ def chunk_iterable(self, iterable, chunk_size):
             yield iterable[i:i + chunk_size]
     
     def process_source(self, source: str, line_start: int = None, line_stop: int = None):
-        split_source = [f'{i:03d} |  {value}' for i, value in enumerate(source.splitlines())]
-        filtered_source = split_source[line_start or 0: line_stop or len(source)]
+        split_source = [f'{i+1:03d} |  {value}' for i, value in enumerate(source.splitlines())]
+
+        if line_start is not None and line_start <= 0:
```
Comment (Jay-Madden) [inline bot/cogs/source_code_cog.py]: this can be 

if line_start and line_start <= 0:
	...

this is called short circuiting

Comment (ellieswaim) [reply bot/cogs/source_code_cog.py]: check the commit after that

**Review by Jay-Madden — APPROVED** [review]

**Pull request merged**